docs/rdataretriever.rst

```diff
@@ -11,7 +11,7 @@ that the ``retriever``'s data handling can easily be integrated into R workflows
 Installation
 ============
 
-To use the R package ``rdataretriever``, you first need to `install the retriever <introduction.html#installing-binaries>`_.
+To use the `R package <https://github.com/ropensci/rdataretriever>`_ ``rdataretriever``, you first need to `install the retriever <introduction.html#installing-binaries>`_.
```

Comment:
Lets as create a variable that we can always call like this example 
https://github.com/weecology/retriever/pull/1454/files#diff-6aa79c70f66ad009e8a4c9f2eca400d7L267

This is called here
https://github.com/weecology/retriever/pull/1454/files#diff-6aa79c70f66ad009e8a4c9f2eca400d7R8
In that way we avoid re defining the same url link several times.

Also I would suggest that we add the link to the word rdataretriever instead of R Package

use the  [R package](https://github.com/ropensci/rdataretriever) rdataretriever  to vs 
use the  R package  [rdataretriever](https://github.com/ropensci/rdataretriever)  to